src/libcharon/plugins/kernel_super/oper_db_client/liboper_db_client/liboper_db_client.h

```diff
@@ -7,27 +7,82 @@ extern "C" {
 
 #include <stdint.h>
 #include <stdbool.h>
-
+#include <stddef.h>
+
+struct liboper_union
+{
+	union
+	{
+		const char *str;
+		uint64_t num;
+		int64_t signed_num;
+		double double_num;
+		int enumurate;
+	};
+	int field_id; /* enum LIBOPER_DB_CLIENT_UNION_FIELD_ID */
```

Comment:
If it's always `enum LIBOPER_DB_CLIENT_UNION_FIELD_ID` then can make the type to be this enum instead of `int` :)